toffy/bin_extraction_test.py

```diff
@@ -0,0 +1,42 @@
+import os
+import tempfile
+import pandas as pd
+import pytest
+from pathlib import Path
+from unittest.mock import patch, call
+
+from toffy import bin_extraction
+from mibi_bin_tools import io_utils
+
+
+@patch('builtins.print')
+def test_extract_missing_fovs(mocked_print):
```

Comment:
This test should include more cases:

1. No FOVs have already been extracted and no Moly points detected (the cleanest run)
2. Some FOVs have already been extracted, no Moly points detected
3. Some FOVs have already been extracted, Moly points detected
4. All FOVs have already been extracted

I'm not sure if the current test structure in `mibi-bin-tools` allows for all of these, it may require adjusting the specific cases.